tests/unit/Trace/RequestHandlerTest.php

```diff
@@ -709,7 +710,13 @@ public function testSetsStatusOfRootSpanOnExitWithHttpResponse()
         $this->assertInstanceOf(SpanData::class, $spanData);
         $this->assertNotEmpty($spanData->endTime());
         $this->assertEquals('main', $spanData->name());
-        $this->assertInstanceOf(Status::class, $spanData->status());
-        $this->assertEquals(200, $spanData->status()->code());
+        $this->assertEquals([Span::ATTRIBUTE_STATUS_CODE => 200], $spanData->attributes());
+
+        if (extension_loaded('opencensus')) {
```

Comment:
Is there a bug in the extension that causes it to ignore status? If so, let's open another bug - the behavior when the extension is installed should be the same.